recipes/dgl/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euxo pipefail
+
+rm -rf build || true
+
+CUDA_CMAKE_OPTIONS="-DCMAKE_CUDA_COMPILER=${CUDA_HOME}/bin/nvcc -DCMAKE_CUDA_HOST_COMPILER=${CXX}"
+
+# Remove -std=c++17 from CXXFLAGS for compatibility with nvcc
+export CXXFLAGS="$(echo $CXXFLAGS | sed -e 's/ -std=[^ ]*//')"
+
+CMAKE_FLAGS="${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release -DPython_EXECUTABLE=${PYTHON}"
+
+echo $CONDA_PREFIX
+
+mkdir build
+cd build
+
+cmake -DUSE_CUDA=ON \
+  -DUSE_CONDA_INCLUDES=ON \
+  -DEXTERNAL_DLPACK_PATH=$PREFIX/include \
+  -DEXTERNAL_DMLC_PATH=$PREFIX/include \
+  -DEXTERNAL_PHMAP_PATH=$PREFIX/include \
+  -DEXTERNAL_NANOFLANN_PATH=$PREFIX/include \
```

Comment:
```suggestion
  -DEXTERNAL_DLPACK_PATH=$BUILD_PREFIX/include \
  -DEXTERNAL_DMLC_PATH=$BUILD_PREFIX/include \
  -DEXTERNAL_PHMAP_PATH=$BUILD_PREFIX/include \
  -DEXTERNAL_NANOFLANN_PATH=$BUILD_PREFIX/include \
```

Reply:
$PREFIX path doesnt containt exterrnal dependencies - it is the installation directory